src/Cardano/Address/Derivation.hs

```diff
@@ -116,13 +117,20 @@ xpubFromBytes :: ByteString -> Maybe XPub
 xpubFromBytes =
     eitherToMaybe . CC.xpub
 
--- | Convert an 'XPub' to a raw 'ByteString'
+-- | Convert an 'XPub' to a raw 'ByteString' (public key plus chain code)
 --
 -- @since 1.0.0
 xpubToBytes :: XPub -> ByteString
 xpubToBytes =
     CC.unXPub
 
+-- | Convert an 'XPub' to a public key 'ByteString' without chain code included
+--
+-- @since 1.0.1
+xpubToBytesWithoutChainCode :: XPub -> ByteString
+xpubToBytesWithoutChainCode (CC.XPub pk _cc) =
+    pk
```

Comment:
What about exposing `getPublicKey` and `getChainCode` instead ? This name doesn't look quite right :no_mouth:

Reply:
done in https://github.com/input-output-hk/cardano-addresses/pull/20/commits/b97b25b7bbf9c5f2130b016a8e88457325063549